src/index.js

```diff
@@ -114,15 +117,27 @@ function exit(signal) {
   }, 500);
 }
 
+async function startService() {
+  if(!await qclient.isConnected()){
+    logger.info('qtum is still starting, please wait');
+    setTimeout(()=>{
+      startService();
+    },1000);
+  }else{
+    startSync();
```

Comment:
I was doing some testing yesterday and I found that `.isConnected()` will come back true before qtumd is fully running. 

Meaning that you should also add a delay when starting all three services.